### PR TITLE
Remove TTS #380

### PR DESCRIPTION
**Pull request by JesterSe7en**
Removes current TTS implementation

### CODE REVIEW

**Comment by shaneknows** [comment]
Not sure if you are working on this, but please merge with dev. I repurposed the bottom app bar class to have the chat link and be setup for the future navigation buttons so please keep that

File: test/command_tutorial_test.dart [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import 'package:flutter_test/flutter_test.dart';
```
Comment (shaneknows) [inline test/command_tutorial_test.dart]: I would recommend just deleting both of these test files since we are no longer pursuing this route.

**Review by shaneknows — COMMENTED** [review]
Is there any reason to keep the speech_to_text.dart file since it doesn't seem like it will be used anywhere anymore? I would say we should also remove the commented items, but since we are re-writing all of those pages anyway it isn't a big deal. I think deleting the empty tests and unused class should be done though since we won't revisit those.

**Review by mdconatser — DISMISSED** [review]

**Comment by shaneknows** [comment]
Fetching mail appears to be broken with this latest push

**Review by mdconatser — APPROVED** [review]

**Review by shaneknows — APPROVED** [review]
Looks good!

**Pull request merged**